### PR TITLE
Adds basic Experience model

### PR DESCRIPTION
**Pull request by enki211**
Also adds filler models for ForeignKey's and ManyToMany's. Registers models with admin site as well.

### CODE REVIEW

File: exdb/models.py [hunk 1]
```diff
@@ -1,3 +1,93 @@
...
+    end_datetime = models.DateTimeField()
+    type = models.ForeignKey(Type)
+    sub_type = models.ForeignKey(SubType)
+    goal = models.TextField(null=True, blank=True)
+    keywords = models.ManyToManyField(Keyword)
+    audience = models.ForeignKey(Audience)
+    guest = models.CharField(max_length=300, null=True, blank=True)
+    guest_office = models.CharField(max_length=300, null=True, blank=True)
+    attendance = models.IntegerField(null=True, blank=True)
+    created_datetime = models.DateTimeField(auto_now_add=True)
```
Comment (shaneallgeier) [inline exdb/models.py]: Using `auto_now_add` is considered a bad practice for most scenarios. In this instance, I think `default=now` would be better. You might also throw a `blank=True` in there for making things easier in the Django admin.

File: exdb/models.py [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+    description = models.TextField()
+    start_datetime = models.DateTimeField()
+    end_datetime = models.DateTimeField()
+    type = models.ForeignKey(Type)
+    sub_type = models.ForeignKey(SubType)
+    goal = models.TextField(null=True, blank=True)
+    keywords = models.ManyToManyField(Keyword)
+    audience = models.ForeignKey(Audience)
+    guest = models.CharField(max_length=300, null=True, blank=True)
+    guest_office = models.CharField(max_length=300, null=True, blank=True)
```
Comment (shaneallgeier) [inline exdb/models.py]: `CharField`s don't really need to be nullable since the empty string is a valid input.

File: exdb/models.py [hunk 3]
```diff
@@ -1,3 +1,93 @@
...
+
+    author = models.ForeignKey(User)
+    planners = models.ManyToManyField(User, related_name='planner_set')
+    title = models.CharField(max_length=300)
+    description = models.TextField()
+    start_datetime = models.DateTimeField()
+    end_datetime = models.DateTimeField()
+    type = models.ForeignKey(Type)
+    sub_type = models.ForeignKey(SubType)
+    goal = models.TextField(null=True, blank=True)
```
Comment (shaneallgeier) [inline exdb/models.py]: `TextField`s don't really need to be nullable since the empty string is a valid input.

File: exdb/models.py [hunk 4]
```diff
@@ -1,3 +1,88 @@
...
+        return self.name
+
+
+class Experience(models.Model):
+    STATUS_TYPES = (
+        ('dr', 'Draft'),
+        ('pe', 'Pending'),
+        ('ap', 'Approval'),
+        ('de', 'Denied'),
+        ('ad', 'Approved'),
```
Comment (tom08) [inline exdb/models.py]: I thought 'Pending' and 'Approval' were supposed to be one status of 'Pending Approval', then when it's approved it just gets the 'Approved' status?

File: exdb/models.py [hunk 5]
```diff
@@ -1,3 +1,88 @@
...
+
+
+class ExperienceComment(models.Model):
+    experience = models.ForeignKey(Experience, related_name='comment_set')
+    message = models.TextField()
+    timestamp = models.DateTimeField(default=now)
+    author = models.ForeignKey(settings.AUTH_USER_MODEL)
+
+    class Meta:
+        ordering = ['timestamp']
```
Comment (tom08) [inline exdb/models.py]: Would we want to order the comments in descending order, so we get the most recent comments listed first?

**Comment by shaneallgeier** [comment]
I approve.

**Comment by tom08** [comment]
I approve.

File: exdb/models.py [hunk 6]
```diff
@@ -1,3 +1,87 @@
...
+    audience = models.ForeignKey(Audience)
+    guest = models.CharField(max_length=300)
+    guest_office = models.CharField(max_length=300)
+    attendance = models.IntegerField(null=True, blank=True)
+    created_datetime = models.DateTimeField(default=now, blank=True)
+    recognition = models.ManyToManyField(Organization)
+    status = models.CharField(max_length=2, choices=STATUS_TYPES, default=STATUS_TYPES[0][0])
+
+    def __str__(self):
+        return self.title
```
Comment (shaneallgeier) [inline exdb/models.py]: This needs to be changed to `self.name`

**Comment by tom08** [comment]
Besides changing the audience max_length to 1, I can't find anything.

**Comment by shaneallgeier** [comment]
I approve.

**Pull request merged**